src/database/query/youfocus.ts

```diff
@@ -0,0 +1,46 @@
+import { Playlist } from '@/database';
+import { DatabaseQueryResponseType, PlaylistModel } from '@/interfaces';
+
+
+const addPlaylistToDB = async ({
+  playlistId,
+  playlistName,
+  description,
+  videos,
+}: PlaylistModel): Promise<DatabaseQueryResponseType> => {
+  try {
+    await Playlist.create({
+      playlistId,
+      playlistName,
+      description,
+      videos,
+    });
+    return { data: { success: true, message: 'Playlist added successfully' } };
+  }
+  catch (error) {
+    return { error };
+  }
+}
```

Comment:
check src/database/query/shiksha.ts

const addACourseToDB = async (
  courseDetails: AddCourseRequestPayloadProps
): Promise<DatabaseQueryResponseType> => {
  try {
    const course = new Course(courseDetails);
    await course.save();
    return { data: course };
  } catch (error) {
    return { error: 'Failed while adding course' };
  }
};

this is how we write code at TBE. What you're doing is something different.